backend/capellacollab/projects/toolmodels/modelsources/git/routes.py

```diff
@@ -11,33 +11,29 @@
 
 from capellacollab.core import database
 from capellacollab.core.authentication import injectables as auth_injectables
-from capellacollab.projects.toolmodels.backups import crud as backups_crud
-from capellacollab.projects.toolmodels.injectables import (
-    get_existing_capella_model,
+from capellacollab.projects.toolmodels import (
+    injectables as toolmodels_injectables,
 )
-from capellacollab.projects.toolmodels.models import DatabaseCapellaModel
-from capellacollab.projects.toolmodels.modelsources.git.models import (
-    DatabaseGitModel,
-    GitModel,
-    PatchGitModel,
-    PostGitModel,
+from capellacollab.projects.toolmodels import models as toolmodels_models
+from capellacollab.projects.toolmodels.backups import crud as backups_crud
+from capellacollab.projects.toolmodels.modelsources.git import (
+    models as git_models,
 )
```

Comment:
why are we not importing this as `from . import models` as it is within the same module?

Reply:
ok here it is referenced that no relative imports should be used https://google.github.io/styleguide/pyguide.html#224-decision

Reply:
Good catch, thanks for finding it! Is fixed.